core/src/test/java/org/elasticsearch/discovery/DiscoveryWithServiceDisruptionsTests.java

```diff
@@ -240,12 +240,16 @@ public void failWithMinimumMasterNodesConfigured() throws Exception {
     /** Verify that nodes fault detection works after master (re) election */
     @Test
     public void testNodesFDAfterMasterReelection() throws Exception {
-        startCluster(3);
+        startCluster(4);
```

Comment:
makes sense, I thought that this was done specifically for this change, but this is just a test improvement.